src/styles/index.scss

```diff
@@ -1,3 +1,18 @@
+@import './utils/variables';
+@import './card';
+@import './star';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: $main-font-size;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 18px;
+}
+
 body {
   margin: 0;
 }
```

Comment:
It's better to move general styling to a separate file to keep the main file clean and clear